.github/workflows/proto-buf-publisher.yml

```diff
@@ -0,0 +1,29 @@
+name: Proto Buf Publishing - Action
+# Protobuf runs buf (https://buf.build/) push updated proto files to https://buf.build/cosmwasm/wasmd
+# This workflow is only run when a .proto file has been changed
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - 'proto/**'
+
+jobs:
+  push:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: bufbuild/buf-setup-action@v0.7.0
+
+      # lint checks
+      - uses: bufbuild/buf-lint-action@v1
```

Comment:
Does this break the build?
We have some linter in the `make proto-all`.

Reply:
It might be redundant, but it just adds inline comments and does not break the build.